examples/nextjs-with-typescript/styles.css

```
@@ -105,6 +105,11 @@ audio {
   margin-bottom: 1rem;
 }
 
+.options {
+  overflow: scroll;
```

```suggestion
  overflow: auto;
```

otherwise it shows even empty scroll bars 😢

---

I guess, though, it's basically always going to scroll. It's scrolling for me and I have a 4K monitor and I run my windows full height.